ext/mini_racer_extension/mini_racer_extension.cc

```diff
@@ -57,6 +57,8 @@ typedef struct {
     ContextInfo* context_info;
     Local<String>* eval;
     useconds_t timeout;
+    size_t max_mem;
+    size_t current_mem;
```

Comment:
I noticed that throughout the codebase, so probably not worth it. Just pointing it out because it is a pet peeve of mine ;p

Basically, the issue is that 64-bit values often must start on a memory address that is a multiple of 64 bits (not true on all platforms, but on some). 32-bit values must always start on a multiple of 32 bits, and so on (generally speaking). 

So, if a struct is laid out as
64 bit
32 bit
64 bit
16 bit
32 bit

the compiler will have to insert padding:

64 bit
32 bit
[waste 32 bits to align the next item]
64 bit
16 bit
[waste 16 bits to align the next item]
32 bit

whereas if it is re-arranged into

64 bit
64 bit
32 bit
32 bit
16 bit
[waste 16 or 48 bits here to get the entire object to be a multiple of 32 or 64 bits so that it can be used in arrays]

you can save some space on the padding. Mostly an issue for things that you have A LOT of, or if the members are laid out pathologically bad. (i.e. char, pointer, char, pointer, char, pointer, ...)